frontend/src/app/components/header/header.component.html

```diff
@@ -1,24 +1,28 @@
 <header class="header">
   <div class="header-title-wrapper">
-    <h1 class="title">РЫБАЧИМ ВМЕСТЕ</h1>
+    <h1 class="title">рыбачим вместе</h1>
     <div class="logo"
       (click)="redirectToMainPage()"
-    >
+      >
       <img 
         [src]="logoPicture"
         [alt]="logoAltImgAttribute"
-      >
+        >
   </div>
   </div>
-  <div class="header-content-wrapper">
+  <div class="header-content-wrapper"
+    >
     <div
       class="burger_menu"
       [ngClass]="{ burger_menu__active: isActiveButtonMenu }"
-      (click)="showSidebar()"
-    ></div>
+      (click)="showSidebarProfile()"
```

Comment:
I don't understand what does it mean "SidebarProfile". I know sidebar entity and i know profile entity but i don't understand how they can be possible together